src/typescript/rollup-plugin-fable/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "rollup-plugin-fable",
+  "version": "0.0.0",
+  "description": "Rollup Plugin for fable-compiler",
+  "main": "index.js",
+  "scripts": {
+    "eslint": "eslint ./"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fable-compiler/Fable.git"
```

Comment:
I'll update. As an aside Babel also uses lerna to manage their monorepo (https://github.com/lerna/lerna). Would that be something we'd like to consider?